### PR TITLE
Update 0.FAQ.md

### PR DESCRIPTION
**Pull request by whitewum**
### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Review by randomJoe211 — APPROVED** [review]

File: docs-2.0/2.quick-start/0.FAQ.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ This topic lists the frequently asked questions for using Nebula Graph. You can
...
 
+## About manual updates
+
+### The behavior of manual is not consistent with the system
+
+Nebula Graph {{ nebula.base20 }} is still under development. Its behavior changes from time to time. Please tell us if the manual and the system are not consistent.
+
+## About forward and backward compatibility
+
+!!! warning "Major version compactibility"
```
Comment (randomJoe211) [inline docs-2.0/2.quick-start/0.FAQ.md:15]: Typo: compatibility

File: docs-2.0/2.quick-start/0.FAQ.md [hunk 2]
```diff
@@ -4,6 +4,24 @@ This topic lists the frequently asked questions for using Nebula Graph. You can
...
+
+    Neubla Graph {{ nebula.release }} is not compactible with Nebula Graph 1.x nor 2.0-RC in both data formats and rpc-protocals, and vise versa. 
+    Check [how to upgrade to Neubla Graph {{ nebula.release }}](../4.deployment-and-installation/3.upgrade-nebula-graph/).
+    You must upgrade [all clients](6.useful-links/).
+
+!!! note "Micro version compactibility"
```
Comment (randomJoe211) [inline docs-2.0/2.quick-start/0.FAQ.md:21]: Typo: compatibility

File: docs-2.0/2.quick-start/0.FAQ.md [hunk 3]
```diff
@@ -4,6 +4,24 @@ This topic lists the frequently asked questions for using Nebula Graph. You can
...
+
+    Neubla Graph {{ nebula.release }} is compactible with Nebula Graph {{ nebula.base20 }} in both data formats and rpc-protocals.
```
Comment (randomJoe211) [inline docs-2.0/2.quick-start/0.FAQ.md:23]: Typo: compatibility and protocol

File: docs-2.0/2.quick-start/0.FAQ.md [hunk 4]
```diff
@@ -4,6 +4,24 @@ This topic lists the frequently asked questions for using Nebula Graph. You can
...
+
+### The behavior of manual is not consistent with the system
+
+Nebula Graph {{ nebula.base20 }} is still under development. Its behavior changes from time to time. Please tell us if the manual and the system are not consistent.
+
+## About forward and backward compatibility
+
+!!! warning "Major version compactibility"
+
+    Neubla Graph {{ nebula.release }} is not compactible with Nebula Graph 1.x nor 2.0-RC in both data formats and rpc-protocals, and vise versa.
```
Comment (randomJoe211) [inline docs-2.0/2.quick-start/0.FAQ.md:17]: protocols
vice versa

**Review by randomJoe211 — CHANGES_REQUESTED** [review]

**Review by cooper-lzy — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Review by cooper-lzy — APPROVED** [review]

**Pull request merged**